lib/jsdom/living/nodes/Node.idl

```diff
@@ -51,4 +51,10 @@ interface Node : EventTarget {
   Node appendChild(Node node);
   Node replaceChild(Node node, Node child);
   Node removeChild(Node child);
+
+  Node? append(sequence<any> nodes);
+  Node? prepend(sequence<any> nodes);
+  Node? after(sequence<any> nodes);
+  Node? before(sequence<any> nodes);
+  Node? replaceWith(sequence<any> nodes);
```

Comment:
In the DOM Standard before/after/replaceWith/remove are defined on `ChildNode`, while prepend/append are defined on `ParentNode`. As such you should edit lib/jsdom/living/nodes/ChildNode.idl and lib/jsdom/living/nodes/ParentNode.idl (and their corresponding implementation files) to add support for these methods.